presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleConnectorTableLayoutHandle.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.plugin.blackhole;
+
+import com.facebook.presto.spi.ConnectorTableLayoutHandle;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+class BlackHoleConnectorTableLayoutHandle
+        implements ConnectorTableLayoutHandle
+{
+    // HACK: dummy field is required to make this class be auto serializable
+    private String dummy;
```

Comment:
If there is no better way to solve this for Jackson, we should make this class an enum.